app/components/Input/input.css

```diff
@@ -24,11 +24,15 @@
 /* size */
 
 .input.large {
-  height: var(--spacing-56);
+  height: 56px;
   font-size: 18px;
```

Comment:
Is it possible to assign the class .large to all components instead of re-writing the typography?

@mruwnik maybe chime in here?

Reply:
Yes, you can just create the .large class in the root.css file and use it universally, but I think you already knew that

Reply:
I think her question was more about adding the `large` class to all components :D 
But yes, it's certainly possible - as long as you specifiy which ones

Reply:
Yes my question is about adding the large class to the components themselves, ideally we would use the typography classes I wrote that once and never write a line of typography again!